addons/modules/functions/fnc_moduleToggleFlashlights.sqf

```diff
@@ -83,6 +84,12 @@ if (!_isAttached) then {
                 [QEGVAR(common,addWeaponItem), [_x, _weapon, selectRandom _flashlights], _x] call CBA_fnc_targetEvent;
             };
 
+            _nvg = hmd _x;
```

Comment:
Missing `private` and could also be moved into `if` after `_removeNVG` is checked.